core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1805,16 +1865,19 @@ class PlaintextConsumerTest extends BaseConsumerTest {
         s"The current assignment is ${consumer.assignment()}")
   }
 
-  @Test
-  def testConsumingWithNullGroupId(): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft", "kraft+kip848"))
+  def testConsumingWithNullGroupId(quorum: String): Unit = {
     val topic = "test_topic"
     val partition = 0;
     val tp = new TopicPartition(topic, partition)
     createTopic(topic, 1, 1)
 
-    TestUtils.waitUntilTrue(() => {
-      this.zkClient.topicExists(topic)
-    }, "Failed to create topic")
+    if (!isKRaftTest()) {
+      TestUtils.waitUntilTrue(() => {
+        this.zkClientOrNull.topicExists(topic)
+      }, "Failed to create topic")
+    }
```

Comment:
I don't know if there's a corresponding way to verify topic existence in KRaft.

Reply:
We could perhaps use `TestUtils.createTopicWithAdmin`. I would take a look at how we migrated other tests to KRaft.

Reply:
Ah, we don't need to check if the topic exists for kraft. In `createTopic`, we already uses `TestUtils.createTopicWithAdmin` which contains checking topic existence.
```
    if (isKRaftTest()) {
      resource(createAdminClient(brokers, listenerName, adminClientConfig)) { admin =>
        TestUtils.createTopicWithAdmin(
          admin = admin,
          topic = topic,
          brokers = brokers,
          controllers = controllerServers,
          numPartitions = numPartitions,
          replicationFactor = replicationFactor,
          topicConfig = topicConfig
        )
      }
    } ...
```